src/ontology/foodon-edit.owl

```diff
@@ -18198,6 +18198,142 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003909 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003909">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001587"/>
+        <obo:IAO_0000115 xml:lang="en">A seed of an almond without skin.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-07-08T20:34:17Z</dc:date>
+        <rdfs:label xml:lang="en">almond meat</rdfs:label>
+    </owl:Class>
+    
+
```

Comment:
Since you have added almond meat, shall we add almond meal as well: https://en.wikipedia.org/wiki/Almond_meal